components/partition_table/parttool.py

```diff
@@ -107,7 +107,7 @@ def main():
         return 1  # nothing found
 
     if args.offset:
-        print('0x%x ' % (found_partition.offset))
+        print('0x%x' % (found_partition.offset))
```

Comment:
This will result in space missing if --offset and --size are both specified. How about something like
```
results = []
if args.offset:
    results.append('0x%x' % (found_partition.offset))
if args.size:
    results.append('0x%x' % (found_partition.size))
print(' '.join(results))
```

Reply:
`print` adds a newline, so it works in that case too